plugins/com.google.cloud.tools.eclipse.appengine.libraries/src/com/google/cloud/tools/eclipse/appengine/libraries/model/CloudLibraries.java

```diff
@@ -64,6 +87,63 @@ public static Library getLibrary(String id) {
     return libraries.get(id);
   }
 
+  private final static Supplier<List<Library>> clientApis = Suppliers.memoize(new Supplier<List<Library>>() {
+    public List<Library> get() {
+      Bundle bundle = FrameworkUtil.getBundle(CloudSdk.class);
+      URL url = bundle.getResource("/com/google/cloud/tools/libraries/libraries.json");
```

Comment:
An alternative URL is `platform:/plugin/com.google.cloud.tools.appengine/com/google/cloud/tools/libraries/libraries.json`

(I wonder if we should rename that bundle to `.cloudsdk`?)

Reply:
Does that URL work without  importing the bundle?

Renaming the bundle sounds like a good idea longterm. We've been talking about renaming appengine-plugins-core anyway. If we do this, we should do it as part of a broader rename to avoid confusion from mismatches.